mlflow/tracking/fluent.py

```diff
@@ -616,6 +616,28 @@ def delete_tag(key: str) -> None:
     MlflowClient().delete_tag(run_id, key)
 
 
+def update_run(status: Optional[str] = None, name: Optional[str] = None) -> None:
+    """
+    Set a new name or status for the current run. If no run is active, this method will create a
+    new active run.
+
+    :param status: The new status of the run to set, if specified.
+                   At least one of ``status`` or ``name`` should be specified.
+    :param name: The new name of the run to set, if specified.
+                 At least one of ``name`` or ``status`` should be specified.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        with mlflow.start_run():
+            mlflow.update_run("FINISHED", "MY_RUN")
+    """
+    run_id = _get_or_start_run().info.run_id
+    MlflowClient().update_run(run_id, status, name)
```

Comment:
@gabrielfu For now, can we remove the fluent API and include only the MlflowClient implementation? My reasoning is that other fluent APIs like `mlflow.start_run()` support setting a name / status and APIs like `mlflow.end_run()` support updating the status. If a user wants to rename a run, they're probably renaming a particular run after it's been completed, rather than an active fluent run.

Reply:
makes sense to me, sure!